hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/SetSpaceQuotaOptions.java

```diff
@@ -32,14 +32,14 @@
   @CommandLine.Option(names = {"--count-quota"},
       description = "For volume this parameter represents the number of " +
```

Comment:
Can you also rename '--count-quota' to '--namespace-quota'?

Reply:
Good point!